app/views/content_items/speech.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, @content_item.title %>
+
```

Comment:
Should we also include:
`<%= content_for :meta_description, @content_item.description %>`
?